### PR TITLE
protocols/gossipsub: Fix a typo in error message

### PR DESCRIPTION
**Pull request by ackintosh**
# Description

Fixed a typo in the error message. 😃 

Please let me know if a changelog entry is needed for this pull request.

## Change checklist

- [x] I have performed a self-review of my own code
- [ ] ~~I have made corresponding changes to the documentation~~
- [ ] ~~I have added tests that prove my fix is effective or that my feature works~~
- [ ] ~~A changelog entry has been made in the appropriate crates~~

### CODE REVIEW

**Review by thomaseizinger — APPROVED** [review]
Nope, changelog not required for this, thank you!

File: protocols/gossipsub/src/behaviour.rs [hunk 1]
```diff
@@ -1528,7 +1528,7 @@ where
                 )
                 .is_err()
             {
-                error!("Failed to send graft. Message too large");
+                error!("Failed to send PRUNE. Message too large");
             }
```
Comment (thomaseizinger) [inline protocols/gossipsub/src/behaviour.rs]: Is "Message to large" really the only failure mode here? That is not obvious from just checking `is_err`?

Comment (ackintosh) [reply protocols/gossipsub/src/behaviour.rs]: You are right. 💡 I'll update this.

**Comment by ackintosh** [comment]
@thomaseizinger Thank you for your review! I have updated this, please have another look when you have time.

**Review by thomaseizinger — APPROVED** [review]
Thanks, this is a good improvement!

**Pull request merged**